cluster/webhookconfigurations/manifests.yaml

```diff
@@ -0,0 +1,26 @@
+---
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingWebhookConfiguration
+metadata:
+  creationTimestamp: null
+  name: validating-webhook-configuration
+webhooks:
+- admissionReviewVersions:
+  - v1
+  clientConfig:
+    service:
+      name: webhook-service
+      namespace: system
+      path: /validate-apiextensions-crossplane-io-v1-compositeresourcedefinition
+  failurePolicy: Fail
+  name: compositeresourcedefinitions.apiextensions.crossplane.io
+  rules:
+  - apiGroups:
+    - apiextensions.crossplane.io
+    apiVersions:
+    - v1
+    operations:
+    - UPDATE
```

Comment:
It could be, but right now there is only immutability check which makes sense only in `UPDATE` requests, so I removed `CREATE` and `DELETE` [here](https://github.com/crossplane/crossplane/pull/2919/files#diff-4d626979bf1caa6537d74a8510da88a56beb463cc1ad4b4b65d49710a09a2e57R36).